sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -294,7 +294,25 @@ src_configure() {
 	fi
 
 	if use cuda; then
-		mycmakeargs+=( -DVTKm_CUDA_Architecture="native" )
+		local MY_CUDA_ARCH=native
+		if [[ -n ${VTK_CUDA_ARCH} ]]; then
+			# we ignore fermi arch, because current nvidia-cuda-toolkit-11*
+			# no longer supports it
+			case ${VTK_CUDA_ARCH} in
+				kepler|maxwell|pascal|volta|turing|ampere|all|none)
```

Comment:
Is there a point to "none"? Sounds like it'd be like USE=-cuda, not that I checked what it does. If users want to disable it, it's the wrong place for this.

Reply:
Haven't checked it yet. Sounds like it is like you're saying. May it be there's some code which builds for cuda without building for specific device?

Reply:
```
# The option 'none' is provided so that when being built as part of another
# project, its own custom flags can be used.
```
I suppose `none` is not relevant for our use case.